### PR TITLE
TRUS-3765b: Changed UTR to Identifier

### PR DESCRIPTION
**Pull request by BriWak**

### CODE REVIEW

File: app/connectors/TrustClaim.scala [hunk 1]
```diff
@@ -23,41 +23,41 @@ import play.api.libs.json.{Json, OFormat}
 
 import scala.language.implicitConversions
 
-case class TrustClaim(utr:String, managedByAgent: Boolean, trustLocked:Boolean)
+case class TrustClaim(identifier:String, managedByAgent: Boolean, trustLocked:Boolean)
```
Comment (adamconder) [inline app/connectors/TrustClaim.scala:26]: @BriWak Can you make sure the response from trusts-store aligns to this and also returns the field identifier in it's response?

File: app/controllers/LogoutController.scala [hunk 2]
```diff
@@ -53,7 +53,7 @@ class LogoutController @Inject()(
           "event" -> "signout",
           "service" -> "maintain-a-trust-frontend",
           "userGroup" -> request.user.affinityGroup.toString,
-          "utr" -> request.userAnswers.utr
+          "utr" -> request.userAnswers.identifier
```
Comment (adamconder) [inline app/controllers/LogoutController.scala:56]: We would want to make this URN for a non-tax event?

File: app/repositories/ActiveSessionRepository.scala [hunk 3]
```diff
@@ -53,23 +53,23 @@ class ActiveSessionRepositoryImpl @Inject()(
     options = BSONDocument("expireAfterSeconds" -> cacheTtl)
   )
 
-  private val utrIndex = Index(
-    key = Seq("utr" -> IndexType.Ascending),
-    name = Some("utr-index")
+  private val identifierIndex = Index(
```
Comment (adamconder) [inline app/repositories/ActiveSessionRepository.scala:56]: We would want to drop the old index here on deployment

File: app/repositories/ActiveSessionRepository.scala [hunk 4]
```diff
@@ -53,6 +59,11 @@ class ActiveSessionRepositoryImpl @Inject()(
     options = BSONDocument("expireAfterSeconds" -> cacheTtl)
   )
 
+  private val utrIndex = Index(
```
Comment (adamconder) [inline app/repositories/ActiveSessionRepository.scala]: We don't need this now?

File: app/repositories/ActiveSessionRepository.scala [hunk 5]
```diff
@@ -33,11 +33,17 @@ import scala.concurrent.{ExecutionContext, Future}
...
+                                           )(override implicit val ec: ExecutionContext)
+  extends ActiveSessionRepository
+    with IndexManager
+    with Logging {
 
-  private val collectionName: String = "session"
+  override val collectionName: String = "session"
+
+  override val dropIndexes: Boolean =
+    config.get[Boolean]("microservice.services.features.mongo.dropIndexes")
```
Comment (adamconder) [inline app/repositories/ActiveSessionRepository.scala:46]: Have you implemented this in application.conf?

Comment (BriWak) [reply app/repositories/ActiveSessionRepository.scala:46]: it was already there

**Pull request merged**